src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -263,15 +267,27 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
         };
 
         thisChiller.SizFac = fields.at("sizing_factor").get<Real64>();
-        if (thisChiller.SizFac <= 0.0) thisChiller.SizFac = 1.0;
+        if (thisChiller.SizFac <= 0.0) {
+            thisChiller.SizFac = 1.0;
+        }
 
-        thisChiller.EvapVolFlowRate = fields.at("chilled_water_maximum_requested_flow_rate").get<Real64>();
-        if (thisChiller.EvapVolFlowRate == DataSizing::AutoSize) {
-            thisChiller.EvapVolFlowRateWasAutoSized = true;
+        {
+            auto tmpFlowRate = fields.at("chilled_water_maximum_requested_flow_rate");
+            if (tmpFlowRate == "Autosize") {
+                thisChiller.EvapVolFlowRate = DataSizing::AutoSize;
+                thisChiller.EvapVolFlowRateWasAutoSized = true;
+            } else {
+                thisChiller.EvapVolFlowRate = tmpFlowRate.get<Real64>();
+            }
         }
-        thisChiller.CondVolFlowRate = fields.at("condenser_maximum_requested_flow_rate").get<Real64>();
-        if (thisChiller.CondVolFlowRate == DataSizing::AutoSize) {
-            thisChiller.CondVolFlowRateWasAutoSized = true;
+        {
+            auto tmpFlowRate = fields.at("condenser_maximum_requested_flow_rate");
+            if (tmpFlowRate == "Autosize") {
+                thisChiller.CondVolFlowRate = DataSizing::AutoSize;
+                thisChiller.CondVolFlowRateWasAutoSized = true;
+            } else {
+                thisChiller.CondVolFlowRate = tmpFlowRate.get<Real64>();
+            }
```

Comment:
:+1: I wasn't actually sure whether the input processor would translate the autosize string into the numeric value, this is fine.

Reply:
If you try to get<Real64> on a an "Autosize" string, you get `[json.exception.type_error.302] type must be number, but is string`. (nlohmann's `json` behaves like a variant)

Reply:
@jmarrec are you saying that as the code is right now, the call to get the value is going to fail for autosize inputs?  Or are you just clarifying what was changed?